target/linux/lantiq/image/Makefile

```diff
@@ -205,9 +213,10 @@ ifeq ($(SUBTARGET),xway)
 
 define Device/bt_homehub-v2b
   $(Device/NAND)
+  DEVICE_VENDOR := BT
+  DEVICE_MODEL := Home Hub 2B
```

Comment:
The model is "Home Hub 2", the variant  "Type B". I know, not that obvious. It's the same for all the other Home Hubs in this file.